plugins/modules/crucible.ocm.rest_api_call.py

```diff
@@ -0,0 +1,66 @@
+import urllib
+from urllib import error as error
+import urllib.request
+import ssl
+import json
+from string import Template
+import types
+from ansible.module_utils.basic import AnsibleModule
+#from module_utils.init.http_request import http_post, http_request
+from ansible.module_utils._text import to_text
+from ansible.module_utils.urls import open_url
+from ..module_utils.http_request import __http_request
+from requests import HTTPError
+import http.client
+from requests.structures import CaseInsensitiveDict
+
+
+def __api_call(module):
```

Comment:
I don't see what benefit this has over putting that logic into main. The parameters are doing to be fairly specific to the module anyway.